index.ts

```diff
@@ -4,5 +4,23 @@
 import 'fast-text-encoding';
 import 'react-native-get-random-values';
 import '@ethersproject/shims';
-// import '@thirdweb-dev/react-native-adapter';
+
+// Override 'atob' to fix base64 decoding issues on Android
+if (typeof atob === 'undefined') {
+  global.atob = function (input) {
+    return Buffer.from(input, 'base64').toString('binary');
+  };
+}
+
+// If 'atob' exists but has issues, override it
+const _atob = global.atob;
+
+global.atob = (val) => {
+  try {
+    return _atob(val);
+  } catch (e) {
+    return _atob(val + '=');
+  }
+};
```

Comment:
Why is this necessary again?

Reply:
Im guessing this is fix the android bug right, but what was not working properly?

Reply:
This fix came from Josh from the privy development team.
> Hey there these issue[s] are related to each other and due to an underlying bug with base 64 encoding and decoding the the android Javascript runtime
> you can override the built in decode method to solve this issue

Reply:
Thanks Josh 🙏